src/components/index.js

```diff
@@ -1,5 +1,8 @@
 // Ex:
 // export * from './my-component'
+export * from "./Button/Button";
```

Comment:
For consistency, can you add `index.js` files to each of these directories so that these are all packaged as named exports? See comment below where `Image` is imported for clarity